commit d5fd57262b9877a0f5b035a7c019d576ee1c60f3
Author: Norman Maurer <nmaurer@redhat.com>
Date:   Wed Aug 6 07:01:24 2014 +0200

    [#2744] Fix flakey HashedWheelTimerTest.testExecutionOnTime()
    
    Motivation:
    
    The calculation of the max wait time for HashedWheelTimerTest.testExecutionOnTime() was wrong and so the test sometimes failed.
    
    Modifications:
    
    Fix the max wait time.
    
    Result:
    
    No more test-failures

diff --git a/common/src/test/java/io/netty/util/HashedWheelTimerTest.java b/common/src/test/java/io/netty/util/HashedWheelTimerTest.java
index 50e1ba6..d8c2a9c 100644
--- a/common/src/test/java/io/netty/util/HashedWheelTimerTest.java
+++ b/common/src/test/java/io/netty/util/HashedWheelTimerTest.java
@@ -131,7 +131,7 @@ public class HashedWheelTimerTest {
     public void testExecutionOnTime() throws InterruptedException {
         int tickDuration = 200;
         int timeout = 125;
-        int maxTimeout = tickDuration + timeout + tickDuration;
+        int maxTimeout = 2 * (tickDuration + timeout);
         final HashedWheelTimer timer = new HashedWheelTimer(tickDuration, TimeUnit.MILLISECONDS);
         final BlockingQueue<Long> queue = new LinkedBlockingQueue<Long>();
 
